### ISSUE TITLE
cnvkit WGS protocol

### ISSUE THREAD

**Issue by AmandaMilou** [chapmanb/bcbio-nextgen#2214]
I read from the cnvkit documentation that there is a WGS protocol available that does not require the input of target and antitarget regions. http://cnvkit.readthedocs.io/en/stable/quickstart.html
However, when I run the bcbio-nextgen pipeline on my WGS data, the target region is set to be transcribed regions (ref-transcripts-transcripts_w10000.bed). Is there a way to use the cnvkit WGS protocol with the bcbio-nextgen pipeline for WGS data? If not, is there a reason that the cnvkit WGS protocol is not used in the bcbio-nextgen pipeline?

**Comment by chapmanb** [comment]
Amanda -- thanks for bringing this up. We swapped to use transcript regions by default because the runtimes for WGS data without target regions were slow, on the order of days. We swapped to use the transcript approach which will give us finer grained resolution in transcript regions and larger scale (anti-target size) resolution in the remainder of the genome. In our experience this captures larger events but they might be more fragmented depending on where they fall. Thanks again for this discussion and hope this works for your needs.
